x/superfluid/keeper/distribution_test.go

```diff
@@ -47,9 +47,20 @@ func (suite *KeeperTestSuite) TestMoveSuperfluidDelegationRewardToGauges() {
 			[]int64{0},
 			[]gaugeChecker{{0, 0, 0, true}},
 		},
+		// In this case, allocate reward to validators with different stat. 
+		// There is no difference between Bonded, Unbonding, Unbonded
 		{
-			"add unbonded validator case",
-			[]stakingtypes.BondStatus{stakingtypes.Bonded, stakingtypes.Unbonded},
+			"add unbonded, unbonding validator case",
+			[]stakingtypes.BondStatus{stakingtypes.Bonded, stakingtypes.Unbonded, stakingtypes.Unbonding},
+			3,
+			[]superfluidDelegation{{0, 0, 0, 1000000}, {1, 1, 0, 1000000}, {2, 2, 0, 1000000}},
+			[]int64{0,1,2},
+			[]gaugeChecker{{0, 0, 0, true}, {1, 1, 0, true}, {2, 2, 0, true}},
```

Comment:
Sorry I don't quite understand this. Why are we expecting a reward in the unbonding case here, but in the next test we do not expect a reward for the same unbonding case? I'm not saying you are wrong, it's just this is unclear to me.

Reply:
In the next test, I want to show that If there is no reward to the validator, the gauge will not be distributed.

Reply:
Ohh, the `rewardVals` field confused me, I now understand